CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+### Added
+
+- Added `sensor` field to `OpLog`.
+- Added the `load_connection_by_prefix_timestamp_key` function and
+  `TimestampKeyExtractor` trait to enable querying of keys prefixed with
+  `timestamp`.
+
```

Comment:
I think it's worthwhile to include the change (or perhaps benefit) this PR will bring to users in the changelog. One key change I see is that the `opLogRawEvents` GraphQL API no longer requires `agentId`; it is now optional. Additionally, the API response will now include logs from all agents, presented in a time-sequential order, rather than being restricted to a single agent's logs.

Reply:
I will supplement the contents in CHANGELOG.

Reply:
Could you wrap code in backticks please?

Reply:
It's done.